arch/arch.h

```diff
@@ -36,6 +36,13 @@ extern unsigned long arch_flags;
 
 #define ARCH_CPU_CLOCK_WRAPS
 
+#define atomic_inc(p, v)					\
```

Comment:
Please rename this macro into atomic_add() since the implementation adds a value to an atomic counter instead of incrementing that counter by one.